### PR TITLE
Refactor UI

### LINKED ISSUE #47: Refactor UI
Refactor common used UI.

### PR DESCRIPTION
**Pull request by PatricioSaldivar**

### CODE REVIEW

File: lib/UI/common.dart [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:chrome_management_app/models/keys_util.dart';
+import 'package:flutter/material.dart';
+
+/// Widget to show a custom text line.
+Widget textLine(
```
Comment (pinkfish) [inline lib/UI/common.dart]: You could make this an actual class, a bit nicer than doing an inline function.

You can do this.

class TextLine extends Padding {
  TextLine({Key key, String test, TextStyle style, EdgeInsetsGeometry geometry) : super(
 padding: padding,
  child: Text(text, maxLines: 2, key: key, overflow: TextOverflow.ellipsis, style: style); }

Just as easy and a class is way easier to deal with.  Same for the rest of these methods.

Comment (pinkfish) [reply lib/UI/common.dart]: I would do this as methods if you are inside a class, but top level global a class makes more sense.

Comment (PatricioSaldivar) [reply lib/UI/common.dart]: Sure, I just did the change, thanks for the advise.

**Review by abbiePetch — APPROVED** [review]
Looks much better! Thanks for refactoring the UI code

File: lib/controllers/widget/hardware_os_card.dart [hunk 2]
```diff
@@ -47,359 +48,92 @@ class _HardwareAndOsCardState extends State<HardwareAndOsCard>
...
+                      DetailedText(
+                          title: 'Device ID',
+                          value: widget._device.deviceId ?? '-'),
+                      DetailedText(
+                          title: 'Last Enrollment Time',
+                          value: widget._device.lastEnrollmentTime ?? '-'),
+                      DetailedText(
+                          title: 'Auto Update Expiration',
+                          value:
+                              '${DateFormat.yMMMd().format(DateTime.fromMillisecondsSinceEpoch(int.parse(widget._device.autoUpdateExpiration)))}'),
```
Comment (Diego1149) [inline lib/controllers/widget/hardware_os_card.dart:130]: What happens if this is not present? Shouldn't we also have the conditional '-' for this?

**Review by Diego1149 — APPROVED** [review]

**Pull request merged**